src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IncentiveReviewsService.kt

```diff
@@ -76,4 +89,21 @@ class IncentiveReviewsService(
 
   private fun calcTypeCount(caseNoteUsage: List<CaseNoteUsage>): Int =
     caseNoteUsage.map { it.numCaseNotes }.fold(0) { acc, next -> acc + next }
+
+  private suspend fun getNextReviewDatesForOffenders(bookingIds: Set<Long>): Map<Long, IepResult> {
```

Comment:
This method will probably need to get or receive the list of Offender Search prisoners for things like the DoB, ACCT, etc

Reply:
Yes. It's probably worth mentioning that I haven't included a branch for the `isIncentivesDataSourceOfTruth` and we're _only_ getting data from the Incentives DB because I've assumed that migration will have taken place by the time this screen is deployed

Reply:
Yeah good point, I noticed that and it makes sense as new table/endpoint is assumed to only work off incentives DB data 👌 (and it's nice and simpler without "weird" branches :))